src/SprykerEco/Zed/Heidelpay/Business/Adapter/Basket/Basket.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Heidelpay\Business\Adapter\Basket;
+
+use Generated\Shared\Transfer\HeidelpayBasketRequestTransfer;
+use Generated\Shared\Transfer\HeidelpayBasketResponseTransfer;
+use Heidelpay\PhpBasketApi\Object\Authentication;
+use Heidelpay\PhpBasketApi\Object\Basket as HeidelpayBasket;
```

Comment:
I would name it `HeidelpayBasketObject` to make this implicitly custom thingy, and not some real model class

Reply:
We will check this in future release.